.github/workflows/release.yml

```diff
@@ -10,12 +10,14 @@ on:
           - patch: for fixes with non-breaking changes (bumps 0.0.X)
           - minor: for new features with breaking changes (bumps 0.X.0)
           - major: for bumping major version with breaking changes (bumps X.0.0)
+          - prerelease (unstable or experimental changes 0.0.0-alpha.X)
         required: true
         type: choice
         options:
           - patch (fixes with non-breaking changes 0.0.X)
           - minor (features with breaking changes 0.X.0)
           - major (major version bump for breaking changes X.0.0)
+          - prerelease (unstable or experimental changes 0.0.0-alpha.X)
```

Comment:
@mrafnadeem-apimatic Does the pre-release code had to be merged in the main branch before running this action?